utils/python/CIME/XML/namelist_defaults.py

```diff
@@ -0,0 +1,113 @@
+"""Interface to `namelist_defaults.xml`.
```

Comment:
I think what you are suggesting is outside the scope of what we had asked
Sean to do. No xml schema changes were allowed in this implementation.

On Thu, Aug 11, 2016 at 7:52 AM, jedwards4b <notifications@github.com>
wrote:

> In utils/python/CIME/XML/namelist_defaults.py
> <https://github.com/ESMCI/cime/pull/377#discussion_r74425431>:
>
> > @@ -0,0 +1,113 @@
> > +"""Interface to `namelist_defaults.xml`.
>
> Following up on my previous comment, this file should inherit from EntryID
> and the xml should follow the entryid format.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ESMCI/cime/pull/377/files/8221c3b6b655fad45d3cb227136384e3407cb8db#r74425431>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHlxExZpAIxrCrhzpqKn8LmuFYMlh65yks5qeykcgaJpZM4Jhszo>
> .
>